src/extractors/root-extractor.js

```diff
@@ -206,6 +206,18 @@ const RootExtractor = {
       type: 'url_and_domain',
     }) || { url: null, domain: null };
 
+    const extendedResults = {};
+    if (extractor.extend) {
+      Object.keys(extractor.extend).forEach(t => {
```

Comment:
This is dumb but the rest of the codebase uses `Reflect.ownKeys` so... I guess let's stick with that here.